src/environments/environment.ts

```diff
@@ -5,7 +5,7 @@
 export const environment = {
   production: false,
   firebase: {
-    
+
   },
```

Comment:
Just remove this line change
Otherwise everything is fine

Reply:
ok